oap-server/server-core/src/test/resources/component-libraries.yml

```diff
@@ -173,8 +176,26 @@ Elasticsearch:
 transport-client:
   id: 48
   languages: Java
-Undertow:
+http:
   id: 49
+  languages: Java,C#,Node.js
```

Comment:
Why change this file?

Reply:
thiis file is sync from oap-server/server-bootstrap/src/main/resources/component-libraries.yml. Maybe this pr should not make too many changes to the file?

Reply:
> Why change this file?

Do I need to reset this file(just change what I need to change)?

Reply:
This is just a UT related file. You just need to change the one in the src/main/resources folder.